Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -0,0 +1,124 @@
+import Library
+import Prelude
+import UIKit
+
+final class PledgeAmountCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var amountInputView: AmountInputView = { AmountInputView(frame: .zero) }()
+  private lazy var inputStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var label: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var stepper: UIStepper = { UIStepper(frame: .zero) }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = self
+      |> \.accessibilityElements .~ [self.label, self.stepper, self.amountInputView]
+      |> \.backgroundColor .~ UIColor.hex(0xf0f0f0) // Extract to Colors.json
+
+    self.contentView.addSubviewConstrainedToEdges(self.rootStackView)
+
+    self.rootStackView.addArrangedSubview(self.label)
+    self.rootStackView.addArrangedSubview(self.inputStackView)
+    self.inputStackView.addArrangedSubview(self.stepper)
+    self.inputStackView.addArrangedSubview(UIView(frame: .zero))
+    self.inputStackView.addArrangedSubview(self.amountInputView)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+
+    let traitCollection = self.traitCollection
+
+    if previousTraitCollection?.preferredContentSizeCategory
+      != traitCollection.preferredContentSizeCategory {
+      self.configureView(for: traitCollection)
+    }
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.inputStackView
+      |> inputStackViewStyle
+
+    _ = self.label
+      |> labelStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.stepper
+      |> stepperStyle
+  }
+
+  // MARK: - Configuration
+
+  private func configureView(for traitCollection: UITraitCollection) {
+    let isAccessibilityCategory = self.traitCollection.ksr_isAccessibilityCategory()
+
+    let alignment: UIStackView.Alignment = (isAccessibilityCategory ? .leading : .center)
+    let axis: NSLayoutConstraint.Axis = (isAccessibilityCategory ? .vertical : .horizontal)
+    let distribution: UIStackView.Distribution = (isAccessibilityCategory ? .equalSpacing : .fill)
+
+    _ = self.inputStackView
+      |> \.alignment .~ alignment
+      |> \.axis .~ axis
+      |> \.distribution .~ distribution
+      |> \.spacing .~ (isAccessibilityCategory ? Styles.grid(1) : 0)
```

Comment:
I noticed that if you type a very long number in the `AmountInputView` that it goes right up to the right of the stepper. Should maybe keep the spacing at `Styles.grid(1)` even when the axis is horizontal?

![image](https://user-images.githubusercontent.com/3156796/56050731-a1682280-5d1a-11e9-9164-3a9ad7c1c669.png)

Reply:
I completely forgot about this..thanks for bringing it up...I'll see if I can use the separator and it's min width or any other option. It actually seems to behave slightly different on device vs SIM.